docs/source/write_first_app.rst

```diff
@@ -101,889 +97,485 @@ bringing up a new one.
-on the required number of peers, gather the chaincode endorsed results from those peers,
-and finally submit the transaction to the ordering service.
+  1. Endorse the transaction proposal by successfully executing the transaction function on sufficient endorsing peers
+     to satisfy the endorsement requirements.
+  2. Submit a transaction containing the proposal and endorsements to the ordering service to be included in a block.
+  3. Wait for the transaction to be distributed to peers and confirm that the transaction is committed successfully,
+     updating ledger state.
 
-Sample application ``'InitLedger'`` call
+Sample application ``InitLedger`` call:
 
-.. code:: bash
+.. code-block:: TypeScript
 
-  // Initialize a set of asset data on the channel using the chaincode 'InitLedger' function.
-  // This type of transaction would only be run once by an application the first time it was started after it
-  // deployed the first time. Any updates to the chaincode deployed later would likely not need to run
-  // an "init" type function.
-  console.log('\n--> Submit Transaction: InitLedger, function creates the initial set of assets on the ledger');
   await contract.submitTransaction('InitLedger');
-  console.log('*** Result: committed');
-
-Chaincode ``'InitLedger'`` function
-
-.. code:: bash
-
-   async InitLedger(ctx) {
-        const assets = [
-            {
-                ID: 'asset1',
-                Color: 'blue',
-                Size: 5,
-                Owner: 'Tomoko',
-                AppraisedValue: 300,
-            },
-            {
-                ID: 'asset2',
-                Color: 'red',
-                Size: 5,
-                Owner: 'Brad',
-                AppraisedValue: 400,
-            },
-            {
-                ID: 'asset3',
-                Color: 'green',
-                Size: 10,
-                Owner: 'Jin Soo',
-                AppraisedValue: 500,
-            },
-            {
-                ID: 'asset4',
-                Color: 'yellow',
-                Size: 10,
-                Owner: 'Max',
-                AppraisedValue: 600,
-            },
-            {
-                ID: 'asset5',
-                Color: 'black',
-                Size: 15,
-                Owner: 'Adriana',
-                AppraisedValue: 700,
-            },
-            {
-                ID: 'asset6',
-                Color: 'white',
-                Size: 15,
-                Owner: 'Michel',
-                AppraisedValue: 800,
-            },
-        ];
-
-        for (const asset of assets) {
-            asset.docType = 'asset';
-            await ctx.stub.putState(asset.ID, Buffer.from(JSON.stringify(asset)));
-            console.info(`Asset ${asset.ID} initialized`);
-        }
-    }
 
-The terminal output entry should look similar to below:
+An initialization function to set up initial ledger state would typically be called only once after a smart contract
```

Comment:
meaning any smart contract is called, the first time this happens, for any smart contract on the channel. ... as opposed to after each s.c. is called for its first invocation.